validator/sawtooth_validator/consensus/poet1/poet_enclave_simulator/poet_enclave_simulator.py

```diff
@@ -343,7 +343,7 @@ def create_wait_timer(cls,
                         previous_certificate_id,
                         cls._seal_private_key))
 
-            tagd = float(struct.unpack('L', tag[-8:])[0]) / (2**64 - 1)
+            tagd = float(struct.unpack('Q', tag[-8:])[0]) / (2**64 - 1)
```

Comment:
Should be part of a separate commit, which explains the change from 'L' to 'Q' (this has nothing to do with keyword args).